front/SettingsForm.tsx

```diff
@@ -107,6 +111,7 @@ const SettingsForm = () => {
           <button
             className="btn btn-lg btn-primary pull-xs-right"
             type="submit"
+            data-cy="settings-submit"
```

Comment:
The data-cy attribute value for the submit button should be 'settings-submit-button' for better clarity.